src/Uno.UI/UI/Xaml/Window/Implementations/BaseWindowImplementation.cs

```diff
@@ -80,7 +80,7 @@ public virtual void Initialize()
 
 	public virtual void Activate()
 	{
-		if (_isClosed)
+		if (NativeWindowFactory.SupportsMultipleWindows && _isClosed)
```

Comment:
Would there be a way to test this with the samples app?

Reply:
Added a blank manual test sample with repro steps